Gems/QtForPython/Code/Source/Platform/Linux/InitializeEmbeddedPyside2.h

```diff
@@ -38,7 +38,7 @@ namespace QtForPython
             void* moduleHandle = dlopen(moduleToLoad, RTLD_NOW | RTLD_GLOBAL);
             if (!moduleHandle)
             {
-                const char* loadError = dlerror();
+                [[maybe_unused]] const char* loadError = dlerror();
                 AZ_Error("QtForPython", false, "Unable to load python library %s for Pyside2: %s", moduleToLoad,
                          loadError ? loadError : "Unknown Error");
```

Comment:
I dont know the cost of calling dlerror, and this is an error condition, so it is not a big deal
If you want to be a bit more efficient for release, you can wrap the variable with `AZ_ENABLE_TRACING`

Reply:
maybe not worth it since i'm going to request this change to be cherry picked into stabilization